spinalcordtoolbox/gui/centerline.py

```diff
@@ -138,15 +151,15 @@ def _init_controls(self, parent):
         custom_mode.setToolTip('Manually select the axis slice on sagittal plane')
         custom_mode.toggled.connect(self.on_toggle_mode)
         custom_mode.mode = 'CUSTOM'
-        custom_mode.sagittal_title = 'Select an axial slice'
+        custom_mode.sagittal_title = 'Select an axial slice.\n%s' % self.params.subtitle
```

Comment:
Consider using the new style string formatting, like the one below where you use the `.format()`.